docs/users/cli-usage.md

```diff
@@ -265,18 +265,21 @@ Successfully started example-site
 Your application can be reached at: http://example-backdrop-site.ddev.site
 ```
 
-### Magento 1 Quickstart
+### OpenMage (formerly Magento 1) Quickstart
 
-1. Download Magento 1 (not recommended, as it's end-of-life) from [release page](https://github.com/OpenMage/magento-mirror/releases).
-2. Make a directory for it, for example `mkdir ~/workspace/magento1` and change directory to the new directory.
-3. Extract the download, for example `tar -zxf ~/Downloads/magento-mirror-1.9.4.3.tar.gz --strip-components=1`
-4. `ddev config` and accept the defaults.
+1. Download OpenMage from [release page](https://github.com/OpenMage/magento-lts/releases).
+2. Make a directory for it, for example `mkdir ~/workspace/OpenMage` and change directory to the new directory.
+3. `ddev config` and accept the defaults.
+4. (Install sample data - see below)
 5. `ddev start`
 6. Follow the URL to the base site.
 
-You may want the [Magento 1 Sample Data](https://magento.com/tech-resources/download#download1759) for experimentation.
+You may want the [Magento 1 Sample Data](https://raw.githubusercontent.com/Vinai/compressed-magento-sample-data/1.9.1.0/compressed-magento-sample-data-1.9.1.0.tgz) for experimentation:
```

Comment:
I imagine this link won't be a permalink, so could you change it to be how to find it? I assume it's always in github.com/Vinai/Vinai/compressed-magento-sample-data ?

Reply:
This file hasn't changed for 6 years. And I am sure, it won't change.

Maybe, somewhen OpenMage could publish their own sample data. But it's either planned nor does a issue for missing sample data exists.